MAINT: datasets: mark functions as out of scope for array api support (#23392)
scipy/scipy: doc/source/dev/api-dev/array_api.rst

from consideration. If a module or submodule contains no in-scope functions, it
is excluded from the tables. For example, `scipy.spatial.transform` is currently
excluded because it's API contains no functions, but may be included in the future

> when the scope expands to include classes. `scipy.odr` and `scipy.datasets` is excluded
> because its contents are considered out-of-scope.

Reviewer: ```suggestion
when the scope expands to include classes. `scipy.odr` and `scipy.datasets` are excluded
because their contents are considered out-of-scope.
```
